extensions/amp-subscriptions/0.1/platform-store.js

```diff
@@ -242,6 +242,58 @@ export class PlatformStore {
     return this.entitlementDeferredMap_[serviceId].promise;
   }
 
+  /**
+   * Get scoreFactor states for each platform
+   * @return {Promise<!JsonObject>}
+   *
+   * return value looks somethinglike this
+   * {
+   *    isReadyToPay: {
+   *     'subscription-google-com': 1,
```

Comment:
This should be "subscribe-google-com". But do you think maybe the `platform -> score` will be more expected? E.g.

```
{
  'subscribe-google-com':  {
    isReadyToPay: 1,
    supportsViewer: 1,
    ...
  }
}
```

Reply:
Don't feel strongly either way. Easy enough to change. WDYT?

Reply:
I think it depends on whether we'd also do something like `scores.isReadyToPay` resolving to something for "all" platforms. If not - I think `scores.platform.isReadyToPay` seems better to me.